### PR TITLE
pre-kcp to main

### PR DESCRIPTION
**Pull request by MatousJobanek**

### CODE REVIEW

File: hack/preview.sh [hunk 1]
```diff
@@ -138,7 +138,7 @@ APPS=$(kubectl get apps -n openshift-gitops -o name)
 if echo $APPS | grep -q spi; then
   if [ "`oc get applications.argoproj.io spi -n openshift-gitops -o jsonpath='{.status.health.status} {.status.sync.status}'`" != "Healthy Synced" ]; then
     echo Initializing SPI
-    curl https://raw.githubusercontent.com/redhat-appstudio/e2e-tests/${E2E_TESTS_COMMIT_SHA:-pre-kcp}/scripts/spi-e2e-setup.sh | bash -s
+    curl https://raw.githubusercontent.com/redhat-appstudio/e2e-tests/${E2E_TESTS_COMMIT_SHA:-main}/scripts/spi-e2e-setup.sh | bash -s
```
Comment (Michkov) [inline hack/preview.sh:141]: this will not work now, it should be still pointing to pre-kcp

File: .ci/oci-e2e-deployment.sh [hunk 2]
```diff
@@ -77,7 +77,7 @@ function createQuayPullSecrets() {
 function executeE2ETests() {
     # E2E instructions can be found: https://github.com/redhat-appstudio/e2e-tests
     # The e2e binary is included in Openshift CI test container from the dockerfile: https://github.com/redhat-appstudio/infra-deployments/blob/main/.ci/openshift-ci/Dockerfile
-    curl https://raw.githubusercontent.com/redhat-appstudio/e2e-tests/pre-kcp/scripts/e2e-openshift-ci.sh | bash -s
+    curl https://raw.githubusercontent.com/redhat-appstudio/e2e-tests/main/scripts/e2e-openshift-ci.sh | bash -s
```
Comment (Michkov) [inline .ci/oci-e2e-deployment.sh:80]: still should point to pre-kcp

File: .ci/oci-e2e-deployment.sh [hunk 3]
```diff
@@ -91,7 +91,7 @@ function prepareWebhookVariables() {
     export webhook_repositoryFullName=$REPO_OWNER/$REPO_NAME
     export webhook_pullNumber=$PULL_NUMBER
     # Rewrite variables in webhookConfig.yml
-    curl https://raw.githubusercontent.com/redhat-appstudio/e2e-tests/pre-kcp/webhookConfig.yml | envsubst > webhookConfig.yml
+    curl https://raw.githubusercontent.com/redhat-appstudio/e2e-tests/main/webhookConfig.yml | envsubst > webhookConfig.yml
```
Comment (Michkov) [inline .ci/oci-e2e-deployment.sh:94]: still should point to pre-kcp

File: .ci/oci-e2e-deployment.sh [hunk 4]
```diff
@@ -100,7 +100,7 @@ git remote add ${MY_GIT_FORK_REMOTE} https://github.com/redhat-appstudio-qe/infr
 
 # Initiate openshift ci users
 export KUBECONFIG_TEST="/tmp/kubeconfig"
-curl https://raw.githubusercontent.com/redhat-appstudio/e2e-tests/pre-kcp/scripts/provision-openshift-user.sh | bash -s
+curl https://raw.githubusercontent.com/redhat-appstudio/e2e-tests/main/scripts/provision-openshift-user.sh | bash -s
```
Comment (Michkov) [inline .ci/oci-e2e-deployment.sh:103]: still should point to pre-kcp

File: hack/preview.sh (hunk 1 above)
Comment (MatousJobanek) [reply hack/preview.sh]: ah, good catch! reverted in: 30ec897

File: .ci/oci-e2e-deployment.sh (hunk 2 above)
Comment (MatousJobanek) [reply .ci/oci-e2e-deployment.sh]: reverted in: 30ec897

File: .ci/oci-e2e-deployment.sh (hunk 3 above)
Comment (MatousJobanek) [reply .ci/oci-e2e-deployment.sh]: reverted in: 30ec897

File: .ci/oci-e2e-deployment.sh (hunk 4 above)
Comment (MatousJobanek) [reply .ci/oci-e2e-deployment.sh]: reverted in: 30ec897

**Review by Michkov — APPROVED** [review]

**Pull request merged**